spec/controllers/static_contents_controller_spec.rb

```diff
@@ -1,140 +1,198 @@
 require 'rails_helper'
 
-# This spec was generated by rspec-rails when you ran the scaffold generator.
-# It demonstrates how one might use RSpec to specify the controller code that
-# was generated by Rails when you ran the scaffold generator.
-#
-# It assumes that the implementation code is generated by the rails scaffold
-# generator.  If you are using any extension libraries to generate different
-# controller code, this generated spec may or may not pass.
-#
-# It only uses APIs available in rails and/or rspec-rails.  There are a number
-# of tools you can use to make these specs even more expressive, but we're
-# sticking to rails and rspec-rails APIs to keep things simple and stable.
-#
-# Compared to earlier versions of this generator, there is very limited use of
-# stubs and message expectations in this spec.  Stubs are only used when there
-# is no simpler way to get a handle on the object needed for the example.
-# Message expectations are only used when there is no simpler way to specify
-# that an instance is receiving a specific message.
-#
-# Also compared to earlier versions of this generator, there are no longer any
-# expectations of assigns and templates rendered. These features have been
-# removed from Rails core in Rails 5, but can be added back in via the
-# `rails-controller-testing` gem.
-
 RSpec.describe StaticContentsController, type: :controller do
 
-  # This should return the minimal set of attributes required to create a valid
-  # StaticContent. As you add validations to StaticContent, be sure to
-  # adjust the attributes here as well.
-  let(:valid_attributes) {
-    skip("Add a hash of attributes valid for your model")
-  }
+  let(:static_content) { FactoryBot.create(:static_content) }
+
+  describe "GET #index" do
```

Comment:
we should check your linting setup.
if i auto lint this file at my machine i get a lot of changes

```
spec/controllers/static_contents_controller_spec.rb | 71 ++++++++++++++++++++++++++++++-----------------------------------------
 1 file changed, 30 insertions(+), 41 deletions(-)
```